controllers/btpoperator_controller_test.go

```diff
@@ -792,20 +773,83 @@ func populateServiceBindingFields(object *unstructured.Unstructured) {
 	Expect(unstructured.SetNestedField(object.Object, "test-service-binding-secret", "spec", "secretName")).To(Succeed())
 }
 
-func clearWebhooks() error {
-	mutatingWebhook := &admissionregistrationv1.MutatingWebhookConfiguration{}
-	if err := k8sClient.DeleteAllOf(ctx, mutatingWebhook, labelFilter); err != nil {
-		if !k8serrors.IsNotFound(err) {
-			return err
+func filterWebhooks(file []byte) (filtered []byte, hasWebhook bool) {
+	lines := strings.Split(string(file), "\n")
+	var buffer []byte
+	isWebhook := false
+	for _, l := range lines {
+		buffer = append(buffer, []byte(l+"\n")...)
+		if l == "---" && len(buffer) != 0 {
+			if isWebhook {
+				split := strings.Split(string(buffer), "\n")
+				// hack for one case where helm templating block spans across two adjacent documents
```

Comment:
there is this section in btp-operator template, [where helm templating `if` block spans across multiple documents](
https://github.com/kyma-incubator/sap-btp-service-operator/blob/6c1becdc5c2a93deffe4e90416e4dd1dabcca618/chart/templates/webhook.yml#L38-L39). This hack covers this particular case.